doc/classes/NetworkedMultiplayerPeer.xml

```diff
@@ -50,10 +50,10 @@
 		</method>
 	</methods>
 	<members>
-		<member name="refuse_new_connections" type="bool" setter="set_refuse_new_connections" getter="is_refusing_new_connections">
+		<member name="refuse_new_connections" type="bool" setter="set_refuse_new_connections" getter="is_refusing_new_connections" default="true">
```

Comment:
Should be checked, documented default differs from generated one.

Reply:
I think this happens because the default value is different for different subclasses. Here, the one from the GDNative peer was probably taken.

Reply:
I'm not sure how we should handle such case. If the base implementation has no default value per se (as there is no boolean behind it, and the methods are virtual and implemented), I guess it's a situation where no default value should be retrieved? i.e. we shouldn't check subclasses and just abort if the base class is inconclusive.

CC @Faless

Reply:
Well, I assume most `NetworkedMultiplayerPeer` will have the default value of `false` to this (with the exception of GDNative when it has no interface). We can change that value there if we want (have if fail to `false` with no interface).

As Akien said, I'm not sure how useful are defaults for properties based on virtual methods, if you need to have a default there it would make more sense having it in the subclass but that would mean changing the current doc quite a lot.

Reply:
Defaults on virtual classes are useful for stuff like BaseButton. Not sure how to mark properties which depend on virtual methods, as there aren't many such properties...